lib/irb/input-method.rb

```diff
@@ -270,6 +274,11 @@ def initialize
       end
     end
 
+    def completion_info
+      autocomplete_message = IRB.conf[:USE_AUTOCOMPLETE] ? 'Autocomplete' : 'Tab Complete'
```

Comment:
Let's use `Reline.autocompletion` as the source of truth here? Otherwise this message would change with `IRB.conf[:USE_AUTOCOMPLETE]` while the actual autocompletion setting will not.